### ISSUE TITLE
NR-Strategy: is "KeepSystemConstantDuringIterations" a duplicate of "RebuildLevel"

### ISSUE THREAD

**Issue by philbucher** [KratosMultiphysics/Kratos#4452]
While debugging the problems of the beams I noticed the setting `KeepSystemConstantDuringIterations` in the NR-Strategy.

It is only being used once [here](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h#L734)
To me it looks like a duplicate of the rebuild level

It seems to be copy-pasted around a couple of times but I couldn't find real usages

What do you think?

**Comment by RiccardoRossi** [comment]
The original idea is to use a secant method, in which RHS is rebuilt at each step but the LHs is computed only once

**Comment by philbucher** [comment]
Ok thx
Should we make this available for the user or not/leave it as is?

**Comment by RiccardoRossi** [comment]
to be honest i am not sure if this works so i am really afraid of exposing this

**Comment by philbucher** [comment]
ok then I guess we leave it as is, thx for the explanation :)
